# Description the point on a trajectory

PathPoint info
float64 v
float64 a
float64 t